### PR TITLE
Add a few tests to the field formatter checking multiline normalization

### PR DESCRIPTION
**Pull request by tobiasdiez**
Inspired by the old (and fixed) bug [SF #1191](http://sourceforge.net/p/jabref/bugs/1191/).

### CODE REVIEW

File: src/test/java/net/sf/jabref/exporter/layout/LatexFieldFormatterTests.java [hunk 1]
```diff
@@ -27,6 +27,21 @@ public void setUp() {
     }
 
     @Test
+    public void normalizeNewlineInAbstractField() {
+        String fieldName = "abstract";
+        String text = "lorem" + Globals.NEWLINE + " ipsum lorem ipsum\nlorem ipsum lorem ipsum\r\ntest";
```
Comment (simonharrer) [inline src/test/java/net/sf/jabref/exporter/layout/LatexFieldFormatterTests.java]: theoretically, `\r` was also used for line breaks on MAC OS before OS-X.

File: src/test/java/net/sf/jabref/exporter/layout/LatexFieldFormatterTests.java [hunk 2]
```diff
@@ -41,6 +56,20 @@ public void preserveNewlineInAbstractField() {
     }
 
     @Test
+    public void preserveMultipleNewlinesInAbstractField() {
+        String fieldName = "abstract";
+        String text = "lorem ipsum lorem ipsum\n\nlorem ipsum lorem ipsum\n";
+        // The newlines are normalized according to the globally configured newline setting in the formatter
+        // Therefore, "\n" has to be replaced by that
+        text = text.replaceAll("\n", Globals.NEWLINE);
```
Comment (simonharrer) [inline src/test/java/net/sf/jabref/exporter/layout/LatexFieldFormatterTests.java]: Can you please create the expected text without the replaceAll action? We want to have fast tests, and replaceAll compiles a regular expression internally when used.

**Comment by tobiasdiez** [comment]
@simonharrer Feedback implemented.

**Comment by simonharrer** [comment]
:+1: looks good to me.

**Pull request merged**